tensorflow/core/api_def/base_api/api_def_GenerateBoundingBoxProposals.pbtxt

```diff
@@ -0,0 +1,17 @@
+op {
+  graph_op_name: "GenerateBoundingBoxProposals"
+  summary: "This op produces Region of Interests from given bounding boxes(bbox_deltas) encoded wrtanchors according to eq.2 in arXiv:1506.01497"
+  description: <<END
+      The op selects top `pre_nms_topn` scoring boxes, decodes them with respect to anchors,
+      applies non-maximal suppression on overlapping boxes with higher than
+      `nms_threshold` intersection-over-union (iou) value, discarding boxes where shorter
+      side is less than `min_size`.
+      Inputs:
+      `scores`: A 4D tensor of shape [Batch, Height, Width, Num Anchors] containing the scores per anchor at given postion
```

Comment:
Use the in_arg and out_arg fields to populate inputs and outputs: https://github.com/tensorflow/tensorflow/blob/273981699d92c1aa08ca52af99ee187ce5aaa5e1/tensorflow/core/framework/api_def.proto#L79

Reply:
This file is auto-generated by tensorflow tools. Do you want me to change them? Wouldn't it break next time they are autogenerated again?

Reply:
Done